qdrant_client/qdrant_fastembed.py

```diff
@@ -8,24 +8,22 @@
 from qdrant_client.http import models
 
 try:
-    from fastembed.embedding import DefaultEmbedding
+    from fastembed import TextEmbedding
 except ImportError:
-    DefaultEmbedding = None
+    TextEmbedding = None
 
 SUPPORTED_EMBEDDING_MODELS: Dict[str, Tuple[int, models.Distance]] = {
-    "BAAI/bge-base-en": (768, models.Distance.COSINE),
```

Comment:
we can't just silently remove models which could have already been used by users